.github/workflows/freebsd.yml

```diff
@@ -0,0 +1,40 @@
+name: FreeBSD
+on:
+  push:
+    branches:
+      - master
+    tags:
+      - v[0-9]*
+  pull_request:
+    branches:
+      - master
+  schedule:
+    - cron:  '0 4 * * FRI'
+
+concurrency:
+  group: ${{ github.event_name }}-${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: ${{github.event_name == 'pull_request'}}
+
+jobs:
+  FreeBSD-test:
+    runs-on: ubuntu-latest
+    name: Test VOTCA on FreeBSD
+    steps:
+    - uses: actions/checkout@v4
+    - name: Test in FreeBSD
+      id: test
+      uses: vmactions/FreeBSD-vm@v1.0.7
+      with:
+        usesh: true
+        prepare: |
+         pkg install -y python science/py-ase science/py-h5py devel/py-lxml devel/py-xmltodict science/rdkit shells/bash science/lammps devel/boost-libs science/libecpint textproc/expat2 math/fftw3 science/gromacs science/hdf5 science/libint2 science/libaec science/libxc cmake eigen perl5 pkgconf devel/py-pybind11 devel/py-pytest gsed 
+         ln -s /usr/local/bin/bash /bin/bash
+         ln -s /usr/local/bin/perl /usr/bin/perl
+         ln -s -f /usr/local/bin/gsed /usr/bin/sed
```

Comment:
@yurivict I don't like these hacks. For `perl` and `bash` we can just `/usr/bin/env perl`, which would be better anyhow. (Related to #1125).

For sed the problem is that gnu sed support `-i` without an extension, but bsd sed not.